meilisearch-release-changelog/v1.0.0.md

```diff
@@ -0,0 +1,121 @@
+# v1.0.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+We are proud to announce we are releasing the first major release of Meilisearch! 🎉 This new release, contrary to v0.30.0, focus on stabilization instead of bringing new features. We took the opportunity to make the final touch regarding breaking changes in the CLI and the error handler.
```

Comment:
@maryamsulemani97, I was thinking, I don't feel really happy with my original sentence. Could we make it more "positive"? For example, precise breaking changes are here for the best. Also, maybe should I talk about the new language support?
I will try to do a better suggestion.

Reply:
```suggestion
We are proud to announce we are releasing the first major release of Meilisearch! 🎉

This new release focuses on stabilizing the CLI instead of only introducing new features. We took this opportunity to introduce the final breaking changes to the CLI and the error handler which would make the user experience more intuitive.

Despite our focus on the Meilisearch stabilization, we also took the time to improve the Meilisearch support of the Japanese and Chinese languages!
```

Reply:
```suggestion
We are proud to announce the first major release of Meilisearch! 🎉
	
This new release focuses on stabilizing the CLI instead of only introducing new features. We took this opportunity to introduce final breaking changes to the CLI and error handler to enhance user experience.
	
Despite our focus on stabilizing Meilisearch, we also took the time to improve support for Japanese and Chinese!